static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-sections.js

```diff
@@ -1,9 +1,18 @@
 // @flow
-const sections = ['politics', 'world', 'business', 'commentisfree'];
 
-const getSection = function(section: string): string {
-    const sectionLower = section.toLowerCase();
-    return /news/.test(sectionLower) || sections.includes(sectionLower)
+// these are considered as 'news' section for outbrain purpose.
+const newsSections = ['politics', 'world', 'business', 'commentisfree'];
+
+/* 
+   Returns the 'outbrain' section ('news' or 'default') according
+   to the given Guardian section (could be any guardian section)
+*/
+const getSection = function(guardianSection: string): string {
```

Comment:
If this isn't really giving a section back but a closed set of values, could it return a defined type instead?

Reply:
That'd be great, but this function's fate is to go soon :)